docs/guides/connecting_to_your_data/how_to_choose_between_working_with_a_single_or_multiple_batches_of_data.md

```diff
@@ -0,0 +1,108 @@
+---
+title: How to choose between working with a single or multiple Batches of data
+---
+import Prerequisites from '../connecting_to_your_data/components/prerequisites.jsx'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+
+This guide will help you decide when you should use a single Batch and when it is beneficial to use multiple Batches, instead.  Building on that knowledge, this guide will also explain the principles behind configuring a Datasource's Data Asset to either only allow for a single Batch to be returned when requested, or to potentially include multiple Batches.  Finally, this guide will discuss how to configure Batch Requests to return a specific Batch or subset of Batches when you are requesting data from a Data Asset that has been configured to include multiple Batches, so that you can get the most versatility out of your Datasource configurations.
+
+By the end of this guide, you will know when it will be most beneficial to be working with a single Batch of data, when it will be most beneficial to be working with multiple Batches of data, and what you need to do in order to ensure your Datasource and Batch Request configurations permit returning multiple Batches or ensure the return of only one.
+
+<Prerequisites>
+
+- [Understand the basics of Datasources in the V3 (Batch Request) API](../../reference/datasources.md)
+- [Understand the basics of requesting data with a Batch Requests](./how_to_get_a_batch_of_data_from_a_configured_datasource.md)
+
+</Prerequisites>
+
+## Steps
+
+The steps of this guide will allow you to quickly determine if you will want to use a single Batch or multiple Batches when creating Expectations and performing Validations.  This knowledge will then inform how you configure Data Assets in your Datasources.
+
+### 1. Determine if a single Batch or multiple Batches of data will be most beneficial in your future use cases
+
+#### When to work with a single Batch of data:
+
+In some cases, a single Batch of data is all you need (or all that is supported).  In particular:
+- When you are Validating data, you will need to specify a single Batch of data.  If you provide multiple Batches in a Batch Request, Validations will default to operating on the last Batch in the list.
+- The Runtime Data Connector does not support handling multiple Batches of data.  You will always configure your Data Assets to correspond to at most a single Batch of Data when you are using a Runtime Data Connector.
+- If you want to quickly create some Expectations as part of an early Expectation Suite or exploratory data analysis, a single Batch is likely to be all that you need.
```

Comment:
Something for my edification, but when do we use `RuntimeDataConnector` vs Runtime Data Connector? Would it only be `RuntimeDataConnector` when we are describing code directly?

Reply:
Yes, if we're referring to a code snippet we would want to use the actual class name.  But if we're just talking about it conceptually, we would just use the name of the concept.